packages/testing/README.md

```diff
@@ -89,6 +89,8 @@ test("sign up", async ({ page }) => {
 
 ### Cypress
 
+⚠️ **Using Cypress to test Clerk is still in the experimental stage. Please be aware that there may be limitations and potential issues when using this approach.**
```

Comment:
```suggestion
⚠️ **Please note:** Support for Cypress is still experimental. Be aware that there are limitations and potential issues at this stage. Please open an issue with a minimal reproduction so that these issues can be fixed. Thanks!
```